app/sagas/init.js

```diff
@@ -14,24 +14,21 @@ import Navigation from '../lib/Navigation';
 import database from '../lib/realm';
 import {
 	SERVERS, SERVER_ICON, SERVER_NAME, SERVER_URL, TOKEN, USER_ID
-} from '../constants/userDefaults';
+} from '../constants/credentials';
 import { isIOS } from '../utils/deviceInfo';
 
 const restore = function* restore() {
 	try {
-		let hasMigration;
-		if (isIOS) {
-			yield RNUserDefaults.setName('group.ios.chat.rocket');
-			hasMigration = yield AsyncStorage.getItem('hasMigration');
-		}
+		yield RNUserDefaults.setName(isIOS ? 'group.ios.chat.rocket' : 'chat.rocket.android');
```

Comment:
We need to set it on `share.js` too. Probably is more better if we create a const with suite/context name